tests/provider.integration.test.ts

```
@@ -0,0 +1,90 @@
+ */
+
+// ---------------------------------------------------------------------------
+// Integration tests — preset resolution logic
+// ---------------------------------------------------------------------------
+describe('Provider preset resolution', () => {
+  test('minimax preset resolves base URL without AI_BASE_URL override', () => {
+    // Simulates what happens in ai.ts when AI_PROVIDER=minimax and
+    // AI_BASE_URL is not set.
+    const provider = 'minimax';
+    const preset = PROVIDER_PRESETS[provider];
+    const baseURL = undefined || preset?.baseURL || 'https://openrouter.ai/api/v1';
```

ESLint

[error] Unexpected constant truthiness on the left-hand side of a `||` expression.

(no-constant-binary-expression)